src/views/CreateView.vue

```diff
@@ -0,0 +1,102 @@
+<template>
+  <h1>ലിങ്ക് സൃഷ്ടിക്കാൻ</h1>
+  <div class="main_area">
+    <input type="text" v-model="subdomain" placeholder="എന്ത് കിട്ടും?" />
+    <button @click="submit" class="submit_btn">സൃഷിടിക്കുക</button>
+  </div>
+  <div v-if="newLink" class="new_link_area">
+    <b>{{ newLink }}</b>
+    <p class="copy-btn" @click="copyURL(newLink)">Copy</p>
+  </div>
+</template>
+<script>
+export default {
+  data() {
+    return {
+      subdomain: "",
+      newLink: "",
+    };
+  },
+  methods: {
+    submit() {
+      const slug = this.subdomain
+        .split(" ")
+        .filter((x) => x != "")
+        .join("-");
+      this.newLink = slug + ".കിട്ടും.com";
+    },
+    async copyURL(mytext) {
+      try {
+        await navigator.clipboard.writeText(mytext);
+        alert("Copied");
+      } catch ($e) {
+        alert("Cannot copy");
+      }
```

Comment:
@Jobin-S Better to show some text message rather than an alert. Would you be able to make another PR?

Reply:
text message as toast?

Reply:
doesn't have to be a full advanced toast, just anything other than an alert box will do